pkg/api/roles.go

```diff
@@ -0,0 +1,40 @@
+package api
+
+import (
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+// API related actions
+const (
+	ActionProvisioningReload = "provisioning:reload"
+)
+
+// API related scopes
+const (
+	ScopeServicesAll           = "services:*"
+	ScopeServicesDashboards    = "services:dashboards"
+	ScopeServicesPlugins       = "services:plugins"
+	ScopeServicesDatasources   = "services:datasources"
+	ScopeServicesNotifications = "services:notifications"
+)
+
+// GetFixedRoleRegistrations returns the list of fixed roles and their grants to organization roles
+// ("Viewer", "Editor", "Admin") or "Grafana Admin" that HTTPServer needs
+func (hs *HTTPServer) AddFixedRoleRegistrations() {
```

Comment:
Does this need to be exposed to the outside of package?

Reply:
I'd suggest to rename the method to `registerFixedRoles`, as `add` and `register` carry the same semantics in this case.

Reply:
Fixed following the other discussing and yes, should be private :slightly_smiling_face: